ci/build_scripts/build.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash -x
+
+set -euo pipefail
+
+help() {
+  cat <<EOF
+Compile and build the tedge components and debian packages.
+Cross is automatically used if you are trying to build for a foreign target (e.g. build for arm64 on a x86_64 machine)
+
+Usage:
+    $0 [ARCH] [--use-cross]
```

Comment:
I feel better replacing `$0` with the script name.

Reply:
Though `$0` give the user context on how to call the script and makes it more dynamic if the script is renamed or is located.

For example if someone calls the script like using `./ci/build_scripts/build.sh` then it shows the help exactly how the user is calling the script:

```sh
Usage:
    ./ci/build_scripts/build.sh [ARCH] [--use-cross]
```

vs.

```sh
# Using hardcoded script path.
Usage:
    ./build.sh [ARCH] [--use-cross]
```